ident/types.go

```diff
@@ -43,24 +43,69 @@ type ID interface {
 	Reset()
 }
 
+// TagName represents the name of a timeseries tag.
+type TagName ID
+
+// TagValue represents the value of a timeseries tag.
+type TagValue ID
+
+// Tag represents a timeseries tag.
+type Tag struct {
+	Name  TagName
+	Value TagValue
+}
+
 // IdentifierPool represents an automatic pool of IDs.
 type IdentifierPool interface {
```

Comment:
How do you feel about renaming this to `ident.Pool`?

Reply:
Sure thing.

Reply:
Awesome, done